src/broadcast/listenbrainzlistener/listenbrainzjsonfactory.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <QByteArray>
+
+#include "track/track.h"
+
+namespace ListenBrainzJSONFactory {
+    enum JsonType {NowListening, Single};
```

Comment:
This can become an enum class.
It should be resnsmec to ListenType or something.

Reply:
This is outdated, I don't use this enum anymore since the format is entirely given by the config object.